configs/11.0/packages/python/sources

```diff
@@ -20,15 +20,17 @@
 #
 
 ATSRC_PACKAGE_NAME="Python"
-ATSRC_PACKAGE_VER=3.6.3
+ATSRC_PACKAGE_VER=3.6
```

Comment:
The same comment made with respect to openssl applies to this file, as well.  However, some packages on advance-toolchain have this kind of point-release information, while others don't.  I'm OK with this change, and I would also be OK without it.